### PR TITLE
feat(upgrade): provide a step by step to publish the schema after an upgrade

### PR DESCRIPTION
**Pull request by DayTF**
> issue: CU-86bx50kfc

### CODE REVIEW

File: how-tos/maintain/push-your-new-version-to-production.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+{% endhint %}
+
+
+When upgrading your agent version, it will only be taken into account if the `.forestadmin-schema.json` with the latest version has been pushed.
+
+## Recommended procedure
+
+At Forest Admin, we advise you to start your migration in your development environment:
+
+1. Upgrade your agent in development following the upgrade notes
```
Comment (ghusse) [inline how-tos/maintain/push-your-new-version-to-production.md:14]: 2. start the agent locally

File: how-tos/maintain/upgrade-notes-rails/upgrade-to-v6.md [hunk 2]
```diff
@@ -13,6 +13,10 @@ Please be aware that while Forest Admin make every effort to ensure that our pla
 
 {% endhint %}
 
+{% hint style="info" %}
+Upgraded your agent but changes are not reflecting on your UI ? [Follow this note](../push-your-new-version-to-production.md).
```
Comment (ghusse) [inline how-tos/maintain/upgrade-notes-rails/upgrade-to-v6.md:17]: I think we should always push to this docs, even if there are no visible problem in the UI.

So I would prefer: "Please follow the recommended procedure regarding forestadmin-schema". Or something like that.

File: how-tos/maintain/push-your-new-version-to-production.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Push your new version to production
+
+{% hint style="info" %}
+Forest Admin is using the [`.forestadmin-schema.json`](https://docs.forestadmin.com/developer-guide-agents-nodejs/under-the-hood/forestadmin-schema) file that is present beside your agent to reflect your model definition as well as the agent version.
```
Comment (ghusse) [inline how-tos/maintain/push-your-new-version-to-production.md:4]: We should also here point out to problems that could occur if the schema is not up-to-date.

And also give pointers for people to check that the version of their schema is matching the version of their agent.

**Review by jeffladiray — APPROVED** [review]
lgtm

File: SUMMARY.md [hunk 4]
```diff
@@ -165,6 +165,7 @@
   * [Manage your Forest Admin environments programmatically](how-tos/maintain/manage-your-forest-admin-programmatically.md)
   * [Changing your domain name](how-tos/maintain/changing-your-domain-name.md)
   * [Migrate to the new role system](how-tos/maintain/migrate-to-the-new-role-system.md)
+  * [Push you new version to production](how-tos/maintain/push-your-new-version-to-production.md)
```
Comment (arnaudbesnier) [inline SUMMARY.md:168]: "Push your new version to production"

**Pull request merged**